src/components/g3w-search.js

```diff
@@ -190,22 +212,20 @@ async function doSearch({
       outputs: show && { title: state.title }
     });
 
-    // @since 3.11.0.
-    // In case of search return == 'search' options.return = 'search', it means that a new search panel needs to show
-    if ('search' === state.return) {
-      // in case of return, a structure of search
-      if (Object.keys((data.data[0] || {}).data || {}).length > 0) {
-        //need to eventually close an open result
-        await GUI.closeContent();
-        const opts = (data.data[0] || {}).data;
-        opts.child = true; //set child true
-        //and open a new Search panel
-        new SearchPanel(opts, true)
-      } else {
-        //otherwise, mean the return of search has no values, so we can show an empty results
-        GUI.outputDataPlace(Promise.resolve({ data: [] }));
-        data = [];
-      }
```

Comment:
@volterra79 please show me how to test this

Reply:
@Raruto you need to install cadastre plugin. It use this type of serach. The first search response, return a new search config that is used to build a new search form